src/malli/core.cljc

```diff
@@ -1749,7 +1749,7 @@
                 {:min 1 :max 1}
                 (-regex-min-max child nested?)))))))))
 
-(defn -=>-schema []
+(defn -=>-schema [{:keys [flat-arrow?]}]
```

Comment:
keys with `?` in the end should be predicate functions, so `flat-arrow` would suffice here. https://github.com/bbatsov/clojure-style-guide?tab=readme-ov-file#naming-predicates